### ISSUE TITLE
Tick "DontDrive" on the avatar creators/full body calibrators

### ISSUE THREAD

**Issue by BlueCyro** [Neos-Metaverse/NeosPublic#3733]
### Is your tweak request related to a problem? Please describe.

It's very frustrating when the avatar creator and full body calibrators' grab handle sliders snap to slightly off or undesirable angles when moving them.

### Describe what would you like tweaked

I've left a comment on #3153 detailing my thoughts, but I figure I would make a new issue for better visibility. I'll copy-paste some of what I wrote on that comment to here.

I'm aware this is a duplicate issue, but I would ask for it to not be closed solely based on that principle, or if it is that the other one be re-opened.

I feel it would be perfectly fine to simply tick "DontDrive" on the avatar creator/fullbody calibrator sliders to true. I'm aware that #1270 would inherently fix the issue altogether, but this would fix it more immediately - and also doesn't go against any of neos' philosophies while doing so, nor would it impact future development on the underlying cause. In the event that the underlying issues are solved, this fix could simply be deprecated/reverted with minimal effort (should it need to be for whatever reason) without affecting the end user.

As it stands there's a mod I made to fix the issue with the full body calibrator: https://github.com/RileyGuy/DontDriveMeMad
And one Sox made to fix the issue with the avatar creator: https://github.com/Sox-NeosVR/CompliantAvatarCreator

I feel like we really don't need to wait on the underlying issues if there's already a valid solution that won't impact development down the line. I genuinely feel this is a warranted change that would make many people very happy in the meantime, along with other unrelated improvements to the full body calibrator and avatar creators down the line.

### Describe alternatives you've considered

Currently the alternatives are to use the mods outlined above to fix it permanently, or manually flick the sliders' DontDrive fields to be true, which fixes the issue temporarily.

### Additional context

It's *very* frustrating for not just myself, but the users coming to neos and fumbling with the full body from this issue and the people trying to help them. I ask that this be reconsidered, as the risks posed to future development are minimal and the impact of having those specific sliders set to DontDrive is negligible at most.
